### PR TITLE
Add operation for terms agg alone

### PR DESCRIPTION
**Pull request by nik9000**
Adds operations to the challenge I've been using to test aggregation
performance that test the performance of the terms aggregation alone
without any child aggregations.

### CODE REVIEW

File: noaa/challenges/default.json
Comment (nik9000) [inline noaa/challenges/default.json:346]: We don't run these in the nightlies so the name change is probably not a big deal.

**Review by DJRickyB — APPROVED** [review]
This all LGTM.  Even though this challenge is not referenced, please hold off on merging as it's end-of-week, and these files are used by nightlies

**Comment by dliappis** [comment]
we'll need some backports too for easier maintainability.

**Comment by nik9000** [comment]
> we'll need some backports too for easier maintainability.

I can pull it back to the `7` branch after I merge it if you'd like.

**Review by dliappis — APPROVED** [review]
LGTM. Did some spot checks and the renamed `aggs` works fine.

**Comment by dliappis** [comment]
Backported to `7.2`, `7.1`, `7.0`, `7`.

**Comment by nik9000** [comment]
> Backported to `7.2`, `7.1`, `7.0`, `7`.

Thanks!

**Pull request merged**
